server/base/src/main/java/org/apache/accumulo/server/security/handler/ZKAuthenticator.java

```diff
@@ -54,6 +56,33 @@ public void initialize(ServerContext context) {
     ZKUserPath = Constants.ZROOT + "/" + context.getInstanceID() + "/users";
   }
 
+  /**
+   * Checks stored users and logs a warning containing the ones with outdated hashes.
+   */
+  public boolean hasOutdatedHashes() {
```

Comment:
I considered that too, but kept it like this because we are checking for the outdated format, and while not being outdated technically means it's the current one, the name still feels misleading.